NEWS.md

```diff
@@ -61,12 +61,37 @@ This is a major release which rewrites the core code to be asynchronous.
   ```
 - `theme` and `icons`: `name` and `file` options have been merged into `theme`/`icons`. See above for an example.
 
+- Major icons and whitespace change. Icons are now part of `format` option as a placeholder in blocks where format is customizable.
+  If you've modified `format` and would like to keep the same behaviour (icon, whitespace)
+  you need to update the value. For example,
+  ```toml
+  [[block]]
+  block = "cpu"
+  format = "cpu:{utilization}"
+  ```
+  needs to be changed to:
+  ```toml
+  [[block]]
+  block = "cpu"
+  format = " $icon $utilization "
+  ```
+
+- **battery**: `hide_missing` option is replaced with `missing_format`. You can set `missing_format = ""` to maintain the behavior
```

Comment:
Leaving a comment for myself, naive but quick approach of trying to find changes in fields

```
git diff -w --diff-filter=MA -U0 master | rg "^[-+]    \w+:\s"
```